app/src/main/res/values/strings.xml

```diff
@@ -45,4 +45,27 @@
     <string name="settings">Settings</string>
     <string name="uploading_data">Uploading data</string>
 
+    <!-- Phone wake up notification   -->
+    <string name="on_wake_notification_title">User Identification</string>
+    <string name="on_wake_notification_message">Tap on notification to identify device user</string>
```

Comment:
Can just say "Tap to identify device user"

Reply:
much better, thanks!